libs/components/tiles/src/lib/modules/tiles/tile-dashboard/tile-dashboard.service.ts

```diff
@@ -45,28 +47,38 @@ export class SkyTileDashboardService {
   @Output()
   public dashboardInitialized = new EventEmitter<void>();
 
-  private tileComponents: ComponentRef<any>[];
-
-  private config: SkyTileDashboardConfig;
-
-  private defaultConfig: SkyTileDashboardConfig;
-
-  private columns: QueryList<SkyTileDashboardColumnComponent>;
-
-  private singleColumn: SkyTileDashboardColumnComponent;
-
-  private mediaSubscription: Subscription;
-
-  private settingsKey: string;
-
+  #columns: QueryList<SkyTileDashboardColumnComponent> | undefined;
+  #config: SkyTileDashboardConfig | undefined;
+  #dragulaService: DragulaService;
+  #dynamicComponentService: SkyDynamicComponentService;
+  #mediaSubscription: Subscription | undefined;
+  #mediaQuery: SkyMediaQueryService;
+  #singleColumn: SkyTileDashboardColumnComponent | undefined;
+  #settingsKey: string | undefined;
+  #tileComponents: ComponentRef<any>[] | undefined;
+  #uiConfigService: SkyUIConfigService;
+
+  // TODO: remove @optional tag in a future breaking change
   constructor(
-    private dragulaService: DragulaService,
-    private mediaQuery: SkyMediaQueryService,
-    private uiConfigService: SkyUIConfigService
+    dragulaService: DragulaService,
+    mediaQuery: SkyMediaQueryService,
+    uiConfigService: SkyUIConfigService,
+    @Optional() dynamicComponentService?: SkyDynamicComponentService
   ) {
-    this.bagId = 'sky-tile-dashboard-bag-' + ++bagIdIndex;
+    if (!dynamicComponentService) {
```

Comment:
Was there something about strict mode that requires us to use `SkyDynamicComponentService`?

Reply:
I don't recall. I'm not seeing anything, so I think it was just something we saw that needed doing and did it.